.github/workflows/kubernetes_test.yaml

```diff
@@ -73,32 +73,32 @@ jobs:
           ip route
       - name: Add DNS entry to hosts
         run: |
-          sudo echo "172.18.1.100  github-actions.qhub.dev" | sudo tee -a /etc/hosts
-      - name: Initialize QHub Cloud
+          sudo echo "172.18.1.100  github-actions.nebari.dev" | sudo tee -a /etc/hosts
+      - name: Initialize Nebari Cloud
         run: |
           mkdir -p local-deployment
           cd local-deployment
-          qhub init local --project=thisisatest --domain github-actions.qhub.dev --auth-provider=password
+          nebari init local --project=thisisatest --domain github-actions.nebari.dev --auth-provider=password
```

Comment:
@iameskild How is this working? Did we copy the `qhub-config.yaml` or rename it somewhere?

Reply:
https://github.com/Quansight/qhub/blob/089da20126f28cdd3a54b9f27c4efed09e3d9b75/qhub/cli/_init.py#L80-L86

Reply:
I did this for the time being:
https://github.com/Quansight/qhub/pull/1480/files#diff-322d8009fe7ede1efc69ad7e72d8985e15edb595e0c8588fa5a66936bc053e0cR84

Reply:
nvm, just red it bellow